bootcamp/tutorials/integration/function_calling.ipynb

```diff
@@ -0,0 +1,430 @@
+{
```

Comment:
<p>We can work a bit to formalize these marketing message. E.g. </p><p>Hope you like this story .... Milvus is xxx. If you like this story and our project, please give us a star ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>